tasks/fix-cat2.yml

```diff
@@ -1015,12 +1023,16 @@
       - name: "MEDIUM | RHEL-07-020900 | AUDIT | All system device files must be correctly labeled to prevent unauthorized modification."
         debug:
             msg: "{{ rhel_07_020900_audit.stdout_lines }}"
+        changed_when:
+            - rhel7stig_audit_complex
```

Comment:
Should there be a conditional here to only show changed when `rhel_07_020900_audit.stdout_lines` contains something?

Reply:
agreed